### PR TITLE
docs: update docs about deploy with Physical Machines

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: Starnop <starnopg@gmail.com>

### Ⅱ. Does this pull request fix one issue?

ref #1094

### CODE REVIEW

**Comment by yeya24** [comment]
For non-container deployment, can we provide a systemd file or something?

**Comment by Starnop** [comment]
> For non-container deployment, can we provide a systemd file or something?

In fact, that's is something that I was thinking. I will add an issue for that firstly.

File: docs/user_guide/install_client.md [hunk 1]
```diff
@@ -1,70 +1,128 @@
...
+This topic explains how to install the Dragonfly `dfclient`.
 
-## Installing from the Latest Package
+## Context
 
-You can install from the latest packages we provided.
+Install the `dfclient` in one of the following ways:
 
-1. Download a package of the client.
+- Deploying with Docker: **It is not recommended when you only need `dfget` for file distribution**.
```
Comment (yeya24) [inline docs/user_guide/install_client.md]: Could you please explain this more detailed in the doc?

File: docs/user_guide/install_client.md [hunk 2]
```diff
@@ -1,70 +1,128 @@
...
 
-2. Enter the target directory.
+2. Enter the project directory.
 
     ```sh
     cd Dragonfly
     ```
 
-3. Build `dfdaemon` and `dfget`.
+3. Compile the source code.
```
Comment (yeya24) [inline docs/user_guide/install_client.md]: What's wrong with `build dfdaemon and dfget`?  I think the previous one is better. What about build the executables/binaries?

File: docs/user_guide/install_server.md [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
 
-This topic explains how to install the Dragonfly server with **Golang version**.
-
-**NOTE**: The Golang version supernode is **not ready for production usage**. However, you can use it more easily in your test environment.
+This topic explains how to install the Dragonfly server.
 
 ## Context
 
-Install the SuperNodes in one of the following ways:
+Install the supernodes in one of the following ways:
```
Comment (yeya24) [inline docs/user_guide/install_server.md]: Install supernode in one of the following ways:

File: docs/user_guide/install_server.md [hunk 4]
```diff
@@ -99,41 +114,53 @@ docker run -d --name supernode --restart=always -p 8001:8001 -p 8002:8002 -v /ho
     make build-supernode && make install-supernode
     ```
 
-4. Start the SuperNode.
+### Start the supernode
```
Comment (yeya24) [inline docs/user_guide/install_server.md]: remove the

File: docs/user_guide/install_server.md [hunk 5]
```diff
@@ -99,41 +114,53 @@ docker run -d --name supernode --restart=always -p 8001:8001 -p 8002:8002 -v /ho
...
-    sudo nginx
-    ```
+   ```sh
+   sudo nginx
+   ```
 
 ## After this Task
 
-- After the SuperNode is installed, run the following commands to verify if Nginx and **Supernode** are started, and if Port `8001` and `8002` are available.
+- After the supernode is installed, run the following commands to verify if Nginx and **supernode** are started, and if Port `8001` and `8002` are available.
```
Comment (yeya24) [inline docs/user_guide/install_server.md]: remove the

File: docs/user_guide/install_client.md (hunk 1 above)
Comment (Starnop) [reply docs/user_guide/install_client.md]: I will find a more suitable place to explain it.

**Pull request merged**